recipes/react-ssr/package.json

```diff
@@ -20,10 +20,10 @@
     "babel-loader": "8.0.5",
     "cross-env": "5.2.0",
     "import-fresh": "3.0.0",
-    "webpack": "4.29.0",
+    "webpack": "4.29.1",
```

Comment:
Since the recipe is using `npm` instead of yarn, isn't better to downgrade to `4.28.4` to not get into the problem that many people were having due to dependency hoist (specifically `acorn`)?

Reply:
Yeah, I would do that and document why it's that way so it won't get broken (test would be even better).

Reply:
I guess we could try to parse a dynamic import with acorn, or with webpack... but that is unpractical. How do you suggest @bebraw ?

Reply:
What issue are you talking about? Could provide any explanatory link?

Reply:
https://github.com/webpack/webpack/issues/8656

Reply:
@PlayMa256 Maybe we should fix Acorn version (``"acorn": "^6.0.7"``) as discussed in the issue and document why this has been done? That seems like a decent compromise for npm users.

Reply:
Fair enough!

Reply:
I'm wondering why do we need to worry about that kind of issues? I mean it's just a small example and it works as is.